docs/monorepo.md

```diff
@@ -1,5 +1,4 @@
-Publishing Packages with the Monorepo
-=====================================
+# Publishing Packages with the Monorepo
```

Comment:
We could change this to just _"Publishing with the Monorepo"_ now that we also have `/apps`

Reply:
Did in https://github.com/Automattic/wp-calypso/pull/32137/files#diff-f7cb0098c1437c5ff523953cc67ca583R1